src/main/java/org/myrobotlab/service/interfaces/ServoControl.java

```diff
@@ -401,7 +401,7 @@ public interface ServoControl extends AbsolutePositionControl, EncoderListener,
   void writeMicroseconds(int uS);
 
   // for instance attachment
-  void attachServoController(String sc, Integer pin, Double pos, Double speed);
+  void attachServoController(String sc);

```

Comment:
Should probably have a javadoc even if it's pretty self-explanatory

Reply:
updated ...